src/components/Divider.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Divider = ({color, style, ...other}) => (
+  <hr
+    style={{
+      borderTopColor: 'rgba(0, 0, 0, .12)',
+      ...style,
+    }}
+  />
+)
+Divider.propTypes = {
+  color: PropTypes.string,
+  style: PropTypes.object,
```

Comment:
Separate custom props (color) from regular react props (style) with a line-break--makes the component "interface" more clear.